### DISCUSSION TITLE
Windows Build not working

### DISCUSSION THREAD

**Discussion by Szarikow** [dyc3/steamguard-cli#204] [Q&A] [answered]
Launched steamguard-cli let it generate manifest then dropped my maFiles into steamguard-cli %appdata% folder, now when trying to launch steamguard-cli.exe it just flashes for one second and closes. Am i doing something wrong or it's a bug?

**Comment by dyc3** [comment, marked as answer]
You need to run the app from the command line. There is no gui right now.

To do this:

1. Open Powershell or Command Prompt
2. Use the `cd` command to navigate to the directory where `steamguard-cli.exe` is located
3. You can then run it
```
.\steamguard-cli.exe --help
```

I haven't tested these steps so let me know if they don't work

**Reply by Szarikow** [reply]
Ah i see, now it makes sense. Thanks
